HunterJuneau/dine-and-dash
# User's Order Details View and User's Payments View

## Description

User's can now see their Order's Details and their Payment Details

## How Can This Be Tested?

Head to the Users View. 
Click on see orders. Then click order details. 

Go back to the user's profile. 
Click the 'See Payments' button

## Types of changes
